### DISCUSSION TITLE
Status of vorta-go?

### DISCUSSION THREAD

**Discussion by drekbour** [borgbase/vorta#1066] [General]
Came here investigating why vorta is the biggest memory user on startup of a small system. Saw a mention of PyQt and mention of checking out `vorta-go` which one _would_ naturally expect to be much lower profile.

At point of writing, activity seems to have paused on `vorta-go`. What is the current status?

**Comment by m3nu** [comment]
I did Vorta-Go as experiment to learn about the state of the toolchain around Qt in Go. It's based on the same UI files, but most functionality isn't implemented.

I found that Qt in Go was a bit hard to work with and I couldn't get translations to work at all. Using Go will also limit the group of people able to contribute. So with Borg being based on Python, it makes sense to stick with that for now.

**Comment by drekbour** [comment]
Can't argue with that as a clear representation! Thanks.
